### ISSUE TITLE
Quest: The Reckoning

### ISSUE THREAD

**Issue by Crispybuilt** [dalaranwow/dalaran-wow#810]
**Quest The Reckoning** - *(ID:13047)*

- **Issue present:** The entire roleplay event doesn't work and the following NPCs are not there where are supposed to be spawned. It is an roleplay event which you just have to watch. 

- **How it should work:** Once you talk to Thorim and choose gossip option **|I'm with you, Thorim.|** the event should start. Just as show on the video.

_____
_____

*Thorim says: It is time I put an end to my brother's madness.*
>Thorim walks up and yells in Loken's face

*Thorim yells: Loken!!*
*Thorim yells: Stand up and fight me, you worthless traitor!*
>Loken knocks Thorim away, then jumps up to him.

*Loken says: You seem eager to join your beloved Sif, brother.*
>Thorim unleashes a flurry of attacks, knocking Loken back towards the center of the temple

*Thorim says: Do not dare...*
*Thorim says: Speak...*
*Thorim says: Her name!!*

*Loken yells: ENOUGH!*
*Loken says: Looks like you can still best me in a fair fight, little brother.*
*Loken says: Unfortunately for you.... this fight is anything but fair!*
>Loken incapacitates Thorim with a beam of some sort, trapping him in a tentacle-shaped form, and summons two Runeforged Servants.

*Loken says: Thanks to your little friend, you've left the Temple of Storms, where you were at your strongest. Instead we fight closer to my master's power.*
*Loken says: Minions! Take my brother and his drake to their new dwellings. Make sure the beast makes it alive. I have special plans for the broodmother.*
*Loken says: As for your life, mortal. I will be generous. After all... why would I destroy my most useful servant? I waited for you for weeks inside that Hyldnir mine.*
*Loken says: The shape wasn't mine, of course. And had you not been so reckless, you would've seen past my illusion. But you came through for me, and for that... I must thank you!*

>Loken, Thorim and the remainder of the NPCs teleport away

_______
_______

Once the entire event is over, you get the quest credit for watching it.
Make sure to despawn the both NPCs and reset them after 5 minutes to their original previous position so more players can do it.

**Video of how it should work:** *http://www.youtube.com/watch?v=Jdrbx7aYmtQ*

________
http://www.wowhead.com/quest=13047/the-reckoning#comments
http://www.wowwiki.com/Quest:The_Reckoning

**Comment by brainos1** [comment]
still cannot complete, thorim does not spawn at location. feb 1 2015

**Comment by taralej90** [comment]
So, uhm... Can this be fixed instead of new reports just being closed and nothing being done? This report is 1.5 years old, seriously...

**Comment by Lothloryen** [comment]
Hello there,

This quest shall be corrected after the next update. Remember that we are a small team with a lot of work in our back, please. Complaining will not make it any faster.

We shall inform better about which issues shall be working after the update.

Thank you very much.

**Comment by taralej90** [comment]
"We shall inform better about which issues shall be working after the update." - this will help a lot, thank you.

**Comment by Anecho** [comment]
Broken again. Thorim is not on the bridge, so event can't be started.

**Comment by daxpedda** [comment]
After the update it seems to work. There were some quotes missing and the combat scene wasn't rly that smooth. They were sometimes looking in completely different directions. But the quest is definitly doable.

**Comment by dramahh** [comment]
nope, still can't do this quest. even tryed dropping quest and grab it again, chair is empty therefore cannon start the event :(

**Comment by daxpedda** [comment]
i had the same problem, you need to wait for respawn. Thorim is waiting on the bridge.

**Comment by Vaxsysl** [comment]
I have another issue, I cannot see Loken or Thorim, only Veranus on the bridge, and its hostile to me.

**Comment by daxpedda** [comment]
that was the case for me too, i just waited for respawn and then thorim spawned and the hostile veranus despawned

**Comment by Vaxsysl** [comment]
Yeeah, just as I posted my message after waiting for some time, it respawned.

**Comment by Augur89** [comment]
Another former bugged quest working now!! Thanks, Lothloryen. Every day I find something new fixed and this makes me very happy :-)

And the people complaining should show some gratitude too, it takes time to fix things but you see eventually they get fixed and the leveling experience with any new alt will be better and better in the future.

So have some faith and stop pressuring the development team, please.

**Comment by wwrpg** [comment]
This appears to be working now. I just completed it today. 
Thorim does not spawn at the quest location, but instead on the bridge closer to Uldar. Simply fly along the bridge and you can find him and talk to him, completing the quest as he moves in and engages Loken.

**Comment by Golganeth** [comment]
Works fine, but once someone does the quest, the respawn timer of Thorim in the bridge is too long (1hour+~).
